container_http/app.py

```diff
@@ -56,8 +68,17 @@ def get_parks_endpoint():
         if park_header_value is None:
             raise ValueError("The 'park' header is missing in the request.")
 
-        mock_species_list = ["Alocasia_macrorrhiza", "Philodendron_selloum", "Anthurium_andraeanum", "Calathea_orbifolia", "Monstera_deliciosa"]
-        json_data = jsonify(mock_species_list)
+        postgres_connection = get_postgres_connection()
+        cursor = postgres_connection.cursor()
+
+        cursor.execute("SELECT species_name FROM species_table WHERE park_name = %s", (park_header_value,))
```

Comment:
this does not fit to the database names
For example "species_table" has to be "Plant"